src/Commands/Disable.php

```diff
@@ -11,22 +11,16 @@ class Disable extends TaskCommand
 {
     /**
      * The Command's name
-     *
-     * @var string
      */
     protected $name = 'tasks:disable';
```

Comment:
Should be `protected string $name = 'tasks:disable';`?

Reply:
I had to revert the Command property types because the parent classes aren't typed and it was causing errors. This doc block should be restored.

Reply:
Actually, these are all typed in the parent class so don't need a `@var` tag here. Probably safe to remove the comment altogether.

See https://github.com/codeigniter4/CodeIgniter4/blob/develop/system/CLI/BaseCommand.php

Reply:
@kenjis https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/6258